lib/src/cartesian/components/BarGraphLabels.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Text, type SkFont } from "@shopify/react-native-skia";
+import type { ChartBounds, PointsArray } from "lib/src/types";
+
+export type BarLabelConfig = {
+  position: "top" | "bottom" | "left" | "right";
+  font: SkFont | null;
+  color?: string;
+};
+
+type BarGraphLabelProps = {
+  points: PointsArray;
+  chartBounds: ChartBounds;
+  barWidth?: number;
+  options: BarLabelConfig;
+};
+
+export const BarGraphLabels = ({
+  points,
+  chartBounds,
+  barWidth = 0,
+  options,
+}: BarGraphLabelProps) => {
+  const labelPositions = points.map(({ x, y, yValue }) => {
+    const yText = yValue?.toString() ?? "";
+    const fontWidth = options.font?.measureText(yText).width ?? 1;
```

Comment:
Let's follow @robwalkerco's lead: https://github.com/FormidableLabs/victory-native-xl/pull/330/files#diff-a4e3c389ee08d29507dce4c5146dd4e6ad25abc773c7756512ad907e9751d2bcR121 and use `getGlyphWidths` instead of `measureText`